### PR TITLE
PT-13138 - Fix orderId and transactionId mixup in webhooks

### PR DESCRIPTION
**Pull request by swDennis**

### CODE REVIEW

File: Tests/Functional/WebhookHandler/WebhookResource.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * (c) shopware AG <info@shopware.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SwagPaymentPayPalUnified\Tests\Functional\WebhookHandler;
+
+class WebhookResource
```
Comment (mitelg) [inline Tests/Functional/WebhookHandler/WebhookResource.php:11]: could you rename it to something, so it contains "test" or so? 

so it gets not confused with the https://github.com/shopware5/SwagPaymentPayPalUnified/blob/main/PayPalBundle/Resources/WebhookResource.php

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Tests/Functional/WebhookHandler/WebhookResource.php (hunk 1 above)
Comment (swDennis) [reply Tests/Functional/WebhookHandler/WebhookResource.php]: Done

**Comment by mitelg** [comment]
commit message is wrong

**Review by mitelg — CHANGES_REQUESTED** [review]
changelog entry is missing. 

we missed that also in https://github.com/shopware5/SwagPaymentPayPalUnified/pull/36

**Review by mitelg — APPROVED** [review]

**Pull request merged**